File: SQLite/example/ch01/examples/perl/connect.pl
#!/usr/bin/env perl

use DBI;

print "Drivers: " . join(", ", DBI->available_drivers()), "\n";

my $dbh = DBI->connect("dbi:SQLite:dbname=/tmp/db", "", "", { RaiseError => 1 });
my $sth = $dbh->prepare("SELECT * FROM session");

$sth->execute;

while($row = $sth->fetch) {
    print join(", ", @$row), "\n";
}

$sth->finish;
$dbh->disconnect;

File: SQLite/example/ch01/examples/python/connect.py
#!/usr/bin/env python

from pysqlite2 import dbapi2 as sqlite

con = sqlite.connect("seinfeld.db")
cur = con.cursor()
cur.execute('SELECT * FROM foods LIMIT 10')

row = cur.fetchone()
while row:
    print row[2]
    row = cur.fetchone()
